ui/src/pages/Upload.tsx

```diff
@@ -0,0 +1,226 @@
+import {
```

Comment:
One high level comment is that some of these field are required while others are not.

The non required fields are: author bio, tags and youtube link all other fields are required

I saw things like the youtube link being required so make sure those match what I said above